### PR TITLE
Issue 985 validate email

### PR DESCRIPTION
**Pull request by teleyinex**

### CODE REVIEW

**Comment by therealmarv** [comment]
:cake:

but no automatic merging ;(

File: test/test_web.py [hunk 1]
```diff
@@ -200,6 +200,83 @@ def test_register_post_creates_email_with_link(self, signer, render, queue):
 
 
     @with_context
+    @patch('pybossa.view.account.mail_queue', autospec=True)
+    @patch('pybossa.view.account.render_template')
+    @patch('pybossa.view.account.signer')
+    def test_update_email_validates_email(self, signer, render, queue):
```
Comment (alejandrodob) [inline test/test_web.py]: so I would test here that the email remains the same. then in a different test that when the user clicks on the link provided, he/she gets the email updated as requested

**Comment by alejandrodob** [comment]
ok, I think finally the :cake: is baked

**Comment by therealmarv** [comment]
:cake: from me too

**Pull request merged**